ocfweb/templates/base.html

```diff
@@ -97,30 +97,17 @@
             </div>
         </div>
 
-        <!-- <div class="ocf-status-bar">
+        <div class="ocf-status-bar">
             <div class="container">
                 <p>
-                    {% if not lab_status.force_lab_closed %}
-                        {% if lab_is_open %}
-                            <strong>Lab Currently Open</strong>
-                        {% else %}
-                            <strong>Lab Currently Closed</strong>
-                        {% endif %}
-                        <span class="nowrap">
-                            {% if not lab_is_open and current_lab_hours %}
-                                Hours:
-                            {% endif %}
-                            {{current_lab_hours|lab_hours_time}}
-                            on {% now "l" %} {% lab_hours_holiday holidays %}
-                            <a class="subtle" href="{% url 'doc' 'services/lab' %}">more &raquo;</a>
-                        </span>
-                    {% else %}
-                        <strong>Lab Temporarily Closed</strong>
-                        <a class="subtle" href="{% url 'home' %}">more &raquo;</a>
-                    {% endif %}
+                    <strong>Vote for Student Tech Fund's renewal at ASUC!</strong>
```

Comment:
maybe give the date? seems more actionable IMO

Maybe: Vote for the Student Tech Fund's Renewal during ASUC Elections on April 5-7!